.examples/raspberrypi/.env

```diff
@@ -0,0 +1,12 @@
+APP_KEY=app_key
+APP_URL="raspberrypi.local:8080"
+DB_HOST=db
+
+MAIL_MAILER=smtp
+MAIL_HOST=smtp.domain.com
+MAIL_PORT=587
+MAIL_USERNAME=username
+MAIL_PASSWORD=password
+MAIL_ENCRYPTION=tls
+MAIL_FROM_ADDRESS=email@example.com
+MAIL_FROM_NAME=Monica
```

Comment:
As I understand the value of having a `.env` file, in all examples I prefer to put everything in the `docker-compose.yml` file for simplicity.
Appart from `DB_HOST` variable, all others will be env-specific, and should not be included (you should be able to get the example and run-it, the rest of the configuration is up to the person)

Reply:
Are you ok including all of these variables in the `docker-compose.yml`? I found the exact variables required for the installation challenging to find and I think it would improve the dev UX if we include them directly in the example.